src/Behavior/MillisecondTimestamp.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace ipl\Orm\Behavior;
+
+use DateTime;
+use InvalidArgumentException;
+use ipl\Orm\Contract\PropertyBehavior;
+
+use function ipl\Stdlib\get_php_type;
+
+class MillisecondTimestamp extends PropertyBehavior
+{
+    public function fromDb($value, $key, $context)
+    {
+        if ($value !== null) {
+            if (is_string($value)) {
+                $value = (int) $value;
```

Comment:
Any reason why this does not also cast to float? `toDb()` does, so I think this should as well.